core/src/main/java/hudson/PluginManager.java

```diff
@@ -177,6 +178,31 @@
     /** Custom plugin manager system property or context param. */
     public static final String CUSTOM_PLUGIN_MANAGER = PluginManager.class.getName() + ".className";
 
+    private static final Logger LOGGER = Logger.getLogger(PluginManager.class.getName());
+
+    /**
+     * Time elapsed between retries to check the updates sites.
+     */
+    public static int checkUpdateSleepTimeMillis;
```

Comment:
Why public?

Reply:
As this "variable" is meant to be a constant, we can name it `CHECK_UPDATE_SLEEP_TIME_MILLIS`. The `public` could be removed but please do not add `final` or `private`.

1) The `private` will prevent us to modify it in tests, to ensure its modification is effective, otherwise its behavior cannot be tested (without reflection I mean).
2) The `final` will prevent Script Console to modify its value on the fly (Groovy does not care about `private` but consider the `final`)
3) Yes you can have a `static final` variable that is not yet initialized but only in a subsequent static block.